src/sentry/incidents/endpoints/validators.py

```diff
@@ -94,6 +94,7 @@ class MetricAlertComparisonConditionValidator(NumericComparisonConditionValidato
 class MetricAlertsDetectorValidator(BaseGroupTypeDetectorValidator):
     data_source = SnubaQueryDataSourceValidator(required=True)
     data_conditions = MetricAlertComparisonConditionValidator(many=True)
+    config = serializers.JSONField(required=False)
```

Comment:
should this be calling the validate on the json schema?

Reply:
you know I'm not even sure this is needed, going to remove and hope nothing fails